tests/distributions/moments/test_means.py

```diff
@@ -0,0 +1,273 @@
+        [Exponential, expon, {"lam": 1}, {"scale": 1}],
+        [Gamma, gamma, {"alpha": 4, "beta": 3}, {"a": 4, "scale": 1 / 3}],
+        [Geometric, geom, {"p": 0.1}, {"p": 0.1}],
+        [Gumbel, gumbel_r, {"mu": 2, "beta": 1}, {"loc": 2, "scale": 1}],
+        [HalfNormal, halfnorm, {"sigma": 1}, {"scale": 1}],
+        [HyperGeometric, hypergeom, {"N": 10, "k": 2, "n": 4}, {"M": 10, "n": 2, "N": 4}],
+        [InverseGamma, invgamma, {"alpha": 2, "beta": 2}, {"a": 2, "scale": 2}],
+        [Laplace, laplace, {"mu": 2, "b": 2}, {"loc": 2, "scale": 2}],
+        [Logistic, logistic, {"mu": 2, "s": 1}, {"loc": 2, "scale": 1}],
+        [LogNormal, lognorm, {"mu": 0.3, "sigma": 0.6}, {"scale": np.exp(0.3), "s": 0.6}],
+        [
+            MatrixNormal,
+            matrix_normal,
+            {"mu": np.eye(3), "rowcov": np.eye(3), "colcov": np.eye(3)},
+            {"mean": np.eye(3), "rowcov": np.eye(3), "colcov": np.eye(3)},
+        ],
+        [Moyal, moyal, {"mu": 2, "sigma": 2}, {"loc": 2, "scale": 2}],
+        [Multinomial, multinomial, {"n": 20, "p": np.ones(6) / 6}, {"n": 20, "p": np.ones(6) / 6}],
+        [
+            MvNormal,
+            multivariate_normal,
+            {"mu": np.ones(3), "cov": np.eye(3)},
+            {"mean": np.ones(3), "cov": np.eye(3)},
+        ],
+        [
+            MvStudentT,
+            multivariate_t,
+            {"mu": np.ones(3), "cov": np.eye(3), "nu": 4},
+            {"loc": np.ones(3), "shape": np.eye(3), "df": 4},
+        ],
+        [NegativeBinomial, nbinom, {"n": 10, "p": 0.5}, {"n": 10, "p": 0.5}],
+        [Normal, norm, {"mu": 2, "sigma": 2}, {"loc": 2, "scale": 2}],
+        [Pareto, pareto, {"alpha": 5, "m": 2}, {"b": 5, "scale": 2}],
+        [Poisson, poisson, {"mu": 20}, {"mu": 20}],
+        pytest.param(
+            Rice, rice, {"b": 2, "sigma": 2}, {"b": 2, "scale": 2}, marks=pytest.mark.xfail
+        ),  # Something is wrong with the Rice mean, maybe a Bessel function in pytensor?
+        [SkewNormal, skewnorm, {"mu": 2, "sigma": 2, "alpha": 2}, {"loc": 2, "scale": 2, "a": 2}],
+        [
+            SkewStudentT,
+            jf_skew_t,
+            {"mu": 2, "sigma": 2, "a": 3, "b": 3},
+            {"loc": 2, "scale": 2, "a": 3, "b": 3},
+        ],
+        [StudentT, t, {"mu": 2, "sigma": 2, "nu": 6}, {"loc": 2, "scale": 2, "df": 6}],
+        [
+            Triangular,
+            triang,
+            {"lower": -3, "upper": 2, "c": 1},
+            {"loc": -3, "scale": 5, "c": 4 / 5},
+        ],
+        [Uniform, uniform, {"lower": -3, "upper": 2}, {"loc": -3, "scale": 5}],
+        [VonMises, vonmises, {"mu": 2, "kappa": 2}, {"loc": 2, "kappa": 2}],
+        [Wald, invgauss, {"mu": 2, "lam": 1}, {"mu": 2, "scale": 1}],
+        [Weibull, weibull_min, {"alpha": 2, "beta": 2}, {"c": 2, "scale": 2}],
+    ],
+)
+def test_mean_equal_to_scipy(dist, scipy_equiv, dist_params, scipy_params):
+    rv = dist.dist(**dist_params)
+    pymc_mean = mean(rv).eval()
```

Comment:
Reduce test overhead: 

```suggestion
    mode = Mode(linker="py", optimizer=None)
    pymc_mean = mean(rv).eval(mode=mode)
```